ecosystem/web-wallet/src/pages/Login.tsx

```diff
@@ -83,8 +83,7 @@ function Login () {
             }
           </Box>
         </Center>
-        <Heading textAlign="center">Wallet</Heading>
-        <Text textAlign="center" pb={8} color={secondaryTextColor[colorMode]}>The official Aptos crypto wallet</Text>
```

Comment:
Can we keep this but just have it say `An Aptos crypto wallet`. Looks weird without any subtext